src/SearchBar/SearchBar.js

```diff
@@ -16,7 +16,10 @@ export default function SearchBar({
       variant="outlined"
       sx={{
         alignItems: "center",
-        borderColor: "rgba(0, 0, 0, 0.23)",
+        borderColor: theme =>
+          theme.palette.mode === "light"
+            ? "rgba(0, 0, 0, 0.23)"
```

Comment:
Do we need to hardcode color here? Can we not reference theme? Is theme.palette.divider not the default that we want to keep in light mode?

Reply:
To be honest, I think you could just use theme.palette.divider for both dark and light mode. I imagine this is the default anyway so maybe you can just remove borderColor completely.

I also think your "theme => theme.palette.text.secondary" is not doing what you think. You have put it in a string so it's not actually evaluating. I think theme.palette.text.secondary is actually white in dark mode so what's happening is it's just reverting to default anyway because it doesn't recognise your string. If you did want to reference theme then it shouldn't be inside a string and should just be theme.palette.text.secondary. Anyway, I think removing borderColor and relying on defaults will work for both dark and light mode.